types/ckeditor__ckeditor5-font/src/documentcolorcollection.d.ts

```diff
@@ -1,4 +1,4 @@
-import { ColorDefinition } from '@ckeditor/ckeditor5-ui/src/colorgrid/colorgrid';
```

Comment:
Regarding this line and a similar one at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53896/files#diff-aa192b9222a4a27175307d892c9e18e7347a2c11dfa45167187a31de5cc13881

For some unknown reason (probably some deep dependency), `dtslint` fails to find the `colorgridview` file in `ckeditor5-ui`

```
4> Errors in typescript@4.4 for external dependencies:
4> ../../ckeditor__ckeditor5-font/src/documentcolorcollection.d.ts(1,33): error TS2307: Cannot find module '@ckeditor/ckeditor5-ui/src/colorgrid/colorgridview' or its corresponding type declarations.
```

so I opted to export `ColorDefinition` from the root of `ckeditor5-ui` because, even though it's not there in the upstream, exporting an interface for TS consumers should not cause any harm.